apps/api/src/app/observability/usecases/get-http-logs/get-http-logs.usecase.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@nestjs/common';
+import { ClickHouseService } from '@novu/application-generic';
+import { GetHttpLogsCommand } from './get-http-logs.command';
+import { GetHttpLogsResponseDto } from '../../dtos/get-http-logs-response.dto';
+
+@Injectable()
+export class GetHttpLogs {
+  constructor(private readonly clickhouseService: ClickHouseService) {}
+
+  async execute(command: GetHttpLogsCommand): Promise<GetHttpLogsResponseDto> {
+    const whereClauses: string[] = ['organization_id = {organizationId:String}'];
+    const params: Record<string, unknown> = {
+      organizationId: command.organizationId,
+    };
```

Comment:
We need to do some injection protection here, to make sure we don't allow arbiterat stuff here